pkgs/development/python-modules/watchfiles/default.nix

```diff
@@ -60,6 +60,10 @@ buildPythonPackage rec {
     cd tests
   '';
 
+  postUnpack = ''
+    substituteInPlace source/watchfiles/version.py --replace "VERSION = '0.0.dev0'" "VERSION = '${version}'"
```

Comment:
At first glance it looks like that with an update to [0.15.0](https://github.com/samuelcolvin/watchfiles/releases/tag/v0.15.0) this would not be needed.